lensless/utils/dataset.py

```diff
@@ -1276,6 +1276,7 @@ def __init__(
         flip_lensed=False,
         downsample=1,
         downsample_lensed=1,
+        downsample_background=1,
```

Comment:
You can simply take the same value as `downsample` since they are measured together, and drop this new parameter.